R/tax_unique.R

```diff
@@ -0,0 +1,279 @@
+#' Filter occurrences to unique taxa
```

Comment:
I couldn't call the documentation for the function on my side for some reason.
Maybe an issue on my side, but run devtools::document() if you haven't already.